### PR TITLE
move to multi package structure

### PR DESCRIPTION
**Pull request by jaumard**

### CODE REVIEW

**Comment by jaumard** [comment]
Yeah look like I didn't brake anything lol

**Comment by rrousselGit** [comment]
packages/flutter_hooks/README appears to be broken

And the root README's logo appears to be broken too

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ jobs:
 
     strategy:
       matrix:
+        package:
+          - flutter_hooks
```
Comment (rrousselGit) [inline .github/workflows/build.yml:17]: leftover?

Comment (jaumard) [reply .github/workflows/build.yml:17]: No it's needed :) on next PR I'll add flutter_mobx_hooks in the matrix to have some CI

**Comment by jaumard** [comment]
> packages/flutter_hooks/README appears to be broken
And the root README's logo appears to be broken too

It's broken because links to your repo, and as it's not merged files are 404

**Comment by jaumard** [comment]
packages/flutter_hooks/README since broken indeed I'll check but logo it's because of path not being merged yet

**Comment by jaumard** [comment]
My symlinks where wrong for some reasons... now it's ok! Good catch!

**Comment by rrousselGit** [comment]
Looking good, thanks!

**Pull request merged**